composer.json

```diff
@@ -10,5 +10,14 @@
     "issues": "https://www.drupal.org/project/issues/webform_civicrm",
     "source": "http://cgit.drupalcode.org/webform_civicrm"
   },
-  "require": { }
+  "require": {
+      "drupal/webform": "^5.0"
```

Comment:
Drupal's Composer facade was populating this, which meant the CI was not aware of the dependency. It also was setting it to `drupal/webform: *` which was "whatever version."